docs/source/index.rst

```diff
@@ -40,12 +40,10 @@ The Jupyter notebook
 
 .. toctree::
    :maxdepth: 1
-   :caption: Developer Documentation
+   :caption: Contributor Documentation
 
-   development_intro
```

Comment:
While I am going to use the intro page in as a landing place to guide contributors (which I define broadly and beyond strictly development activities) going forward (to things like translation, outreach materials creation, etc.), I'm fine with it being commented out for now versus removed.